lib/qops/deployment/instances.rb

```diff
@@ -152,7 +152,7 @@ def clean
 
     # Find all instances to be destroyed
     retrieve_instances.each do |instance|
-      next if instance.hostname == 'master'
+      next if instance.hostname == "#{config.hostname_prefix}master"
```

Comment:
or for now if it regex matches `master`

`next if instance.hostname =~ /master/